docs/sources/panels/transformations/types-options.md

```diff
@@ -372,6 +372,8 @@ The available conditions for all fields are:
 - **Regex:** Match a regex expression
 - **Is Null:** Match if the value is null
 - **Is Not Null:** Match if the value is not null
+- **Is Undefined:** Match if the value is undefined
+- **Is Not Undefined:** Match if the value is not undefined
```

Comment:
What does not undefined mean? If it means defined, maybe we can mention: "Is Defined:** Match if the value is defined"?

Reply:
I guess eliminating the double negative makes sense, but in this case `null` will be considered as defined. However, as @ryantxu mentioned already, perhaps it's not even necessary to differentiate between `null` and `undefined` for matchers although they do have semantic differences for other parts of the code.